app.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers");
+
+app.use(express.json());
```

Comment:
We do not need this at this stage and should remove it until we are sending a body with our request 🙂